presidio-anonymizer/Dockerfile

```diff
@@ -10,5 +10,6 @@ RUN pipenv sync
 
 COPY . /usr/bin/${NAME}/
 
-EXPOSE ${FLASK_RUN_PORT}
-CMD pipenv run flask run --host 0.0.0.0
\ No newline at end of file
+# [ADO-2680] Use FLASK_RUN_PORT and flask run --host 0.0.0.0
```

Comment:
A really small remark:
# [TODO ADO-2680] Use FLASK_RUN_PORT and flask run --host 0.0.0.0
Please add the TODO - this way it is being recognized as a TODO and marked accordingly in the code.

Reply:
missed that